### PR TITLE
eslint 0.16.x

### PR DESCRIPTION
**Pull request by jdlehman**
- Update eslint to 0.16.x
- Update eslint-loader to 0.5.0

### CODE REVIEW

**Comment by jdlehman** [comment]
Feel free to just cherry pick the first commit, I was not sure if you wanted to update to 0.5.0, or 0.4.1. It seems that you have been updating the minor version for eslint updates in the past, so I went with that.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "index.js"
   ],
   "peerDependencies": {
-    "eslint": "^0.15.0"
+    "eslint": "^0.16.0"
```
Comment (ericclemmons) [inline package.json]: `peerDependencies` shouldn't be this strict:
> https://docs.npmjs.com/files/package.json#peerdependencies

If this project works on both versions' API, then the range should be used here instead.

Comment (ericclemmons) [inline package.json]: Seeing how the tests pass between both version, we should be able to use a range and support both libraries.

@MoOx Thoughts?

**Comment by MoOx** [comment]
I am working on several improvements for this loader. Hopefully it will be ready tomorrow guys. I am doing my best and got a long day.

File: package.json (hunk 1 above)
Comment (MoOx) [inline package.json]: I will take a look to that tomorrow.

**Comment by ericclemmons** [comment]
@MoOx Hey man, no worries!  Maintaining open-source projects are strenuous, especially with the normal 9-5 grind.

Let us know if there's anything else we can help with :)

**Comment by grassator** [comment]
Yeah, sorry for an extra PR, I was just doing my own open-source maintainer duty today and got carried away...

File: package.json (hunk 1 above)
Comment (MoOx) [inline package.json]: FYI, upcoming version will not work with 0.15 & before.

**Comment by MoOx** [comment]
Closed by #9d52d09a31002f3197cb815813419caa552a30e4

**Comment by MoOx** [comment]
Closed by 9d52d09a31002f3197cb815813419caa552a30e4

**Comment by MoOx** [comment]
[0.5 is out, checkout the changelog](https://github.com/MoOx/eslint-loader/releases/tag/0.5.0)

**Pull request closed without merging**